.circleci/config.yml

```diff
@@ -131,7 +134,6 @@ commands:
             # If that fails, then check for the latest from any branch
             - << parameters.key_prefix >>-{{ .Branch }}
             - << parameters.key_prefix >>-master
-            - << parameters.key_prefix >>
```

Comment:
If `checksum` fails (which never happens), then new cache is created and will be stored. Don't use this fall back, clutter code.